app/src/main/java/com/hyperion/ths/marvel_03/data/source/HeroRepository.java

```diff
@@ -0,0 +1,41 @@
+package com.hyperion.ths.marvel_03.data.source;
+
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.data.source.HeroDataSource;
```

Comment:
ERROR: Redundant import from the same package - com.hyperion.ths.marvel_03.data.source.HeroDataSource.